cwf/gateway/integ_tests/test_runner.go

```diff
@@ -78,6 +79,24 @@ const (
 	GyValidityTime  = 60   // in second
 )
 
+type TrafficOutput struct {
+	End TrafficOutputEnd `json:"end"`
+}
+
+type TrafficOutputEnd struct {
+	SumSent     TrafficSummary `json:"sum_sent"`
+	SumReceived TrafficSummary `json:"sum_received"`
+}
+
+type TrafficSummary struct {
+	Start         float64 `json:"start"`
+	End           float64 `json:"end"`
+	Seconds       float64 `json:"seconds"`
+	Bytes         int     `json:"bytes"`
+	BitsPerSecond float64 `json:"bits_per_second"`
+	Retransmits   int     `json:"retransmits"`
+}
+
```

Comment:
Maybe it would be good to do that in the UEsim itself and return the parsed object on `cwfprotos.GenTrafficResponse`

I forgot to finish the JSON marshall into a go type here
http://github.com/magma/magma/blob/41293699fc2d1a7bdd06fd83f0418282a3a21c33/cwf/gateway/services/uesim/servicers/uesim.go#L351-L351

Reply:
Oooo yes, I can do that there too. Good call.

Reply:
Oh I didn't realize you did half the work on that side already. I'll convert everything to use structs and will return the types protobuf.